src/h5web/vis-packs/core/line/LineVis.tsx

```diff
@@ -87,9 +80,17 @@ function LineVis(props: Props) {
     return domain ? extendDomain(domain, 0.05, scaleType) : DEFAULT_DOMAIN;
   }, [scaleType, domain]);
 
+  const tooltipFormatters = useTooltipFormatters(
+    abscissas,
+    abscissaLabel,
+    dataArray,
+    errorsArray
+  );
+
   return (
     <figure className={styles.root} aria-labelledby="vis-title">
       <VisCanvas
+        canvasTitle={title}
         abscissaConfig={{
           domain: abscissaDomain,
           showGrid,
```

Comment:
Not modified in this PR but the ` isIndexAxis: !abscissaParams.value` below can be replaced by `isIndexAxis: !abscissaValue`